signer/src/storage/postgres.rs

```diff
@@ -133,6 +132,35 @@ struct DepositStatusSummary {
     signers_public_key: PublicKeyXOnly,
 }
 
+/// A convenience struct for retrieving a deposit request report
```

Comment:
`deposit` -> `withdrawal`

Reply:
Fixed in https://github.com/stacks-network/sbtc/pull/1344/commits/25dda0e8e426738c74bcd04365bcc2bcad8c5fea.